plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/connection/TcpConnection.java

```diff
@@ -67,6 +67,7 @@ protected DockerResponse request(String method, String path, String query, List<
         for (Pair<String, ?> header : headers) {
             connection.setRequestProperty(header.first, String.valueOf(header.second));
         }
+        connection.setRequestProperty("Host", url.toString());
```

Comment:
In that case not only host will be put in that header but host + path +query. 
@l0rd Do you think it is OK or only host+port should be put in that header.